internal/ocm/ocm.go

```diff
@@ -108,16 +105,12 @@ func (csc *ClusterServiceClient) PostCSCluster(ctx context.Context, cluster *cmv
 }
 
 // UpdateCSCluster sends a PATCH request to update a cluster in Clusters Service
-func (csc *ClusterServiceClient) UpdateCSCluster(ctx context.Context, internalID InternalID, cluster *cmv1.Cluster) (*cmv1.Cluster, error) {
-	client, ok := internalID.GetClusterClient(csc.Conn)
-	if !ok {
-		return nil, fmt.Errorf("OCM path is not a cluster: %s", internalID)
-	}
-	clusterUpdateResponse, err := client.Update().Body(cluster).SendContext(ctx)
+func (csc *ClusterServiceClient) UpdateCSCluster(ctx context.Context, internalID InternalID, cluster *arohcpv1alpha1.Cluster) (*arohcpv1alpha1.Cluster, error) {
+	clusterUpdateResponse, err := csc.Conn.AroHCP().V1alpha1().Clusters().Cluster(internalID.ID()).Update().Body(cluster).SendContext(ctx)
```

Comment:
I think is better here to make a call to GetClusterClient and have a new function GetAroHcpClusterClient that will give the ARO endpoint.

Reply:
done 👍